e2e-tests/tests/version-service/05-assert.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 500
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  generation: 1
+  name: cluster-latest-mysql
+status:
+  observedGeneration: 1
+  replicas: 3
+  readyReplicas: 3
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  generation: 1
+  name: cluster-latest-orc
+status:
+  observedGeneration: 1
+  replicas: 3
+  readyReplicas: 3
+---
+apiVersion: ps.percona.com/v1alpha1
+kind: PerconaServerMySQL
+metadata:
+  generation: 2
+  name: cluster-latest
+status:
+  mysql:
+    ready: 3
+    size: 3
+    state: ready
+    version: 8.0.28-20
```

Comment:
We can leave this check, but let's check if the images in pods are correct like we do in PSMDB. Because this status can be wrong if there's a bug.

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/151/commits/80dfb98b5d0eecaa473ba7a1b26de810db253e6a